data/korath/korath missions.txt

```diff
@@ -257,3 +257,28 @@ mission "Small Scale Delivery"
 	on complete
 		payment 1000
 		dialog `You find a Korath who seems to have been waiting for you in the spaceport. They greet you by showing their palms, then outstretch their hands as if waiting to receive the package. You hand off the small delivery and they hand you a small payment of <payment>.`
+
+mission "Korath Boarding Response"
+	invisible
+	boarding
+	repeat
+	minor
+	deadline 1
+	to offer
+		random < 5
+		or
+			"reputation: Korath" < -5000
```

Comment:
```suggestion
			"reputation: Korath" < -100
```

I'm actually inclined to just remove this entirely, or simplify it to just "-1". Why should the Korath send a rescue fleet based on how much they hate the player? As this isn't a revenge mission, but a *rescue* mission, any negative reputation is sufficient. After all, it just has to be sufficiently negative that they know that the player isn't sending over a friendly repair crew. 

The random <5 is more than sufficient to keep this as being a rare event. The player will be able, in all likelyhood, to loot 95% of the disabled Korath ships without interruption. 

On the other hand, having this scale could be good. Something like random <5 while reputation is between -1 and -100, goes up to <10 while reputation is between -101 and -1000; then <25 for the next increment, etc. At <-10000 the player gets a major incoming fleet about half the time...

Reply:
I was trying to avoid a massive spike in difficulty level. The sliding scale you propose is a great idea. This will require a few rounds of testing to make sure I get the three-layer-deep "to offer" logic right.

Reply:
You now have a sliding difficulty scale, and the boarding response will happen at a 5% rate regardless of reputation. Also, hunting Korath civilians will get you the high arrival rates much quicker than hunting their military vessels.

Reply:
Doesnt Korath reputation start at -1k btw?

Reply:
No, -10.

Reply:
> I've changed it to `"reputation: Korath" < -10`, so you have to have pissed them off at least a little bit before they respond.
> 
> Also, they don't just take down one person. They clean out the entire system in seconds, making it safe for the raids to proceed, and then return to their home systems. The cost to them is quite small: just a rescue fleet on standby, and maybe a loss of a fighter or two. It won't evolve into a major cost unless the player is mass farming Korath with a fleet that is a threat to four Korath Defense fleets.

And given that the Korath are literally a space-living species, their ships are already predominantly in space. So all that really happens is that a fast response fleet spends a bit of fuel and hangs out in System B instead of system A for a day. Responding to something like this might actually be a net *positive* for them in terms of resource acquisition.